src/org/folio/rest/TenantConfiguration.groovy

```diff
@@ -83,6 +83,22 @@ class TenantConfiguration extends GeneralParameters {
         }
     }
 
+    void worldcatCheck(OkapiTenant tenant){
+        auth.getOkapiToken(tenant, tenant.getAdminUser())
+        String url = okapi_url + "/copycat/profiles/f26df83c-aa25-40b6-876e-96852c3d4fd4"
+        ArrayList headers = [
+            [name: 'Content-type', value: "application/json"],
+            [name: 'X-Okapi-Tenant', value: tenant.getId()],
+            [name: 'X-Okapi-Token', value: tenant.getAdminUser().getToken() ? tenant.getAdminUser().getToken() : '', maskValue: true]
+        ]
+        def res = http.getRequest(url, headers)
+        if (res.status == HttpURLConnection.HTTP_OK) {
+            logger.info("Worldcat exists")
+        } else {
+            logger.info("Worldcat - copycat/profiles/{id} is not exists")
```

Comment:
This function will always execute successfully. I would add `return false` here and change the type of returned by function value.

Reply:
Fixed